### ISSUE TITLE
[Request feature] Calculate grade only in major courses.

### ISSUE THREAD

**Issue by pansitwattana** [tmwatchanan/reg-cmu-cgpa-calculator#6]
It should be a benefit for someone who want to know their major grade.

**Comment by tmwatchanan** [comment]
That is absolutely true. Can we just use the first three numbers in the course number to determine what major courses?

**Comment by pansitwattana** [comment]
We should have dictionary for first three digits for each faculty. For CPE does it count only 261xxx?

**Comment by tmwatchanan** [comment]
Yes. But in the case of ISNE, it looks like both 261xxx and 269xxx are counted.
